### PR TITLE
[RS-161] labels manager

### PR DESCRIPTION
**Pull request by Zyruks**
## Description

This pull request adds the Labels Manager feature to the project.

## Related Issue
#161 

## Motivation and Context

This feature is needed to easily manage multiple labels in the project, making it more organized and efficient.

## How Has This Been Tested

I tested this feature by creating a test repository and adding labels using the Labels Manager. I also tested deleting and editing labels to ensure the feature is working properly.
### Github Action Running
Creating, updating and Deleting Labels
image

## Types of changes

- [ ] Docs change / refactoring / dependency upgrade
- [ ] Bug fix
- [x] New feature
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

- [x] My code follows the code style this project.
- [x] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

**Comment by Zyruks** [comment]
# Labels create it with this configuration.
image
image
image

File: .github/workflows/labels.yml [hunk 1]
```diff
@@ -0,0 +1,29 @@
+name: labeler
+
+on:
```
Comment (serudda) [inline .github/workflows/labels.yml:3]: @Zyruks este yml se ejecutaria siempre que se hace push en main?... No hay una manera que se cree solo al crear el proyecto, o cuando cambie un Label... no se en que nos puede afectar que siempre se este ejecutando este yaml?

Comment (Zyruks) [reply .github/workflows/labels.yml:3]: El action solo se va a ejecutar cuando allá un push en main que modifique uno de los archivos que está en el path. Es decir: 


```yaml
paths:
      - ".github/labels.yml"
      - ".github/workflows/labels.yml"
```

Comment (serudda) [reply .github/workflows/labels.yml:3]: Buenisimo

Comment (Zyruks) [reply .github/workflows/labels.yml:3]: Fue investigado en la documentación de los Github Actions 

image

Y de igual forma fue testeado

**Pull request merged**